### DISCUSSION TITLE
The pdf not render from first page

### DISCUSSION THREAD

**Discussion by Mah0913** [stephanrauh/ngx-extended-pdf-viewer#2419] [Q&A] [answered]
when i try to render the pdf, it always start showing from middle of page instead of first page
![Screenshot 2024-07-05 155645](https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/155726532/75734ba1-4894-4aed-b510-9c06535a549e)
for example this render from pg13
![image](https://github.com/stephanrauh/ngx-extended-pdf-viewer/assets/155726532/f7d557de-90c3-4596-b64f-76ba158b6fe6)

```
<ngx-extended-pdf-viewer [useBrowserLocale]="true" height="100vh" [delayFirstView]="1000"
                        [src]="question.imageUrl" [textLayer]="true" [showPresentationModeButton]="true"
                        [showRotateButton]="false" [showBookmarkButton]="false" [showOpenFileButton]="false"
                        [showDownloadButton]="false" [showPrintButton]="false" [enablePinchOnMobile]="true"
                        [enableDragAndDrop]="false" [enablePrint]="false" [zoom]="'page-fit'"
                        [showPropertiesButton]="false" #pdfViewerr>
</ngx-extended-pdf-viewer>
```
the question.imageUrl is the pdf retrieve from server. Anyone meet this issue before?

**Comment by stephanrauh** [comment, marked as answer]
Oh, you're faster than me - I just took a similar screenshot! You've stumbled upon a convenience feature of pdf.js. I suppose it's useless in an Angular application, so maybe I should remove it. pdf.js always tries to use the old zoom setting, rotation, and page number when you open a PDF file for the second time.

Adding `[page]="1"` fixes the problem.

BTW, `[delayFirstView]` is a very old attribute - you don't need it with a current version of the library.

**Reply by Mah0913** [reply]
Thanks for the reply~ it helping!!
